### ISSUE TITLE
Format section/subsection - missing tabs/spaces.

### ISSUE THREAD

**Issue by botzill** [CenterForOpenScience/pydocx#202]
Hi.

I have such a file:

[subsections_format.docx](https://github.com/CenterForOpenScience/pydocx/files/112134/subsections_format.docx)

after converting to html we get:

![2ea64158-ae83-11e5-8ed4-d56bff32bef1](https://cloud.githubusercontent.com/assets/4686040/12716344/6d35b8d4-c8e8-11e5-9edb-a32d3088b7e5.png)

As you can see the subsections are not properly formatted. If you guide me where to look for this issue I can submit a pull request to solve this. 

Thx a lot.

**Comment by jhubert** [comment]
## Before:

![image](https://cloud.githubusercontent.com/assets/3384/13727369/c5698a32-e8a3-11e5-87cf-993d25b0d6ff.png)


## After:

![image](https://cloud.githubusercontent.com/assets/3384/13727370/da78e2ec-e8a3-11e5-8212-368d24ada99f.png)

**Comment by winhamwr** [comment]
It looks like there is one definite bug, along with some confusion about the styling.

"Gather Items for Re-pricing" should definitely be in the same list as "Prepare your markdown gun" and it's not obvious to me why it isn't.

The first step will be adding a [fixtures testcase](https://github.com/CenterForOpenScience/pydocx/blob/master/tests/export/test_docx.py#L25) by adding both a .docx and .html file in the [fixtures](https://github.com/CenterForOpenScience/pydocx/tree/master/tests/fixtures) directory. That will let us define the input and then the expected output.

If anyone could help with that part, it would be appreciated. From there, someone will need to dive in to the OOXML in the .docx to figure out why we're parsing the .docx as separate lists instead of one list.

**Comment by jhubert** [comment]
I dove in and took a look at the OOXML for this. I've added the fixtures as well.

It looks like what's happening is that it's being considered three different lists because the bulleted list is breaking up the numeric list.

![image](https://cloud.githubusercontent.com/assets/3384/13793728/9724c7c2-eab5-11e5-8010-3d9e987170c0.png)

Here is the _simplified_ relevant `document.xml` OOXML:

```xml
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="1"/></w:numPr></w:pPr><w:r><w:t>one</w:t></w:r></w:p>
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="1"/></w:numPr></w:pPr><w:r><w:t>two</w:t></w:r></w:p>
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="1"/></w:numPr></w:pPr><w:r><w:t>three</w:t></w:r></w:p>
<w:p w:rsidP="007F6A48"><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="5"/></w:numPr><w:tabs><w:tab w:val="clear" w:pos="709"/></w:tabs></w:pPr><w:r><w:t>AAA</w:t></w:r></w:p>
<w:p w:rsidP="007F6A48"><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="5"/></w:numPr><w:tabs><w:tab w:val="clear" w:pos="709"/></w:tabs></w:pPr><w:r><w:t>BBB</w:t></w:r></w:p>
<w:p w:rsidP="007F6A48"><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="5"/></w:numPr><w:tabs><w:tab w:val="clear" w:pos="709"/></w:tabs></w:pPr><w:r><w:t>CCC</w:t></w:r></w:p>
<w:p><w:pPr><w:numPr><w:ilvl w:val="2"/><w:numId w:val="1"/></w:numPr></w:pPr><w:r><w:t>alpha</w:t></w:r></w:p>
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="1"/></w:numPr></w:pPr><w:r><w:t>four</w:t></w:r></w:p>
<w:p/>
<w:p/>
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="2"/></w:numPr></w:pPr><w:r><w:t>xxx</w:t></w:r></w:p>
<w:p w:rsidP="007F6A48"><w:pPr><w:numPr><w:ilvl w:val="1"/><w:numId w:val="6"/></w:numPr></w:pPr><w:r><w:t>yyy</w:t></w:r></w:p>
<w:p/>
<w:p><w:pPr><w:numPr><w:ilvl w:val="0"/><w:numId w:val="3"/></w:numPr></w:pPr><w:r><w:t>www</w:t></w:r></w:p>
<w:p w:rsidP="007F6A48"><w:pPr><w:numPr><w:ilvl w:val="1"/><w:numId w:val="7"/></w:numPr></w:pPr><w:r><w:t>zzz</w:t></w:r></w:p>
```

The full `document.xml` OOXML is beautified in this gist: https://gist.github.com/jhubert/29f7899073b765e74297

**Comment by kylegibson** [comment]
Can you include `numbering.xml` and `styles.xml` as well?

**Comment by jhubert** [comment]
Of course. Gist update: https://gist.github.com/jhubert/29f7899073b765e74297

Also, here is the docx file:
[nested_multitype_lists.docx](https://github.com/CenterForOpenScience/pydocx/files/174828/nested_multitype_lists.docx)

**Comment by jhubert** [comment]
@kylegibson I'm about to hire someone to work on this issue. Have you already started?

**Comment by kylegibson** [comment]
Hi Jeremy. None of us have started work on this issue. I expect it will be awhile before we have time to dedicate to fixing this. We'll be happy to review any PRs that you submit!

**Comment by jhubert** [comment]
Awesome. Good to know. We'll get a PR together. :)

**Comment by jhubert** [comment]
The fix that @botzill put together in #225 is now live in production. No issues so far. 💯
